ci-operator/step-registry/telco5g/origin/tests/telco5g-origin-tests-commands.sh

```diff
@@ -44,30 +41,236 @@ git clone -b "${CNF_BRANCH}" "${CNF_REPO}" cnf-features-deploy
+        dest: '/tmp/kcli_${CLUSTER_NAME}_openshift-tests'
+
+    - name: Copy list of tests to bastion host
+      ansible.builtin.copy:
+        src: '${ARTIFACT_DIR}/${TEST_FILE}'
+        dest: '/tmp/kcli_${CLUSTER_NAME}_${TEST_FILE}'
+
+    - name: Copy script to run openshift-tests binary to installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli scp '/tmp/kcli_${CLUSTER_NAME}_run-openshift-tests.sh' \\
+            'cloud-user@${CLUSTER_NAME}-installer:/home/cloud-user/run-openshift-tests.sh' \\
+          && kcli ssh 'cloud-user@${CLUSTER_NAME}-installer' 'chmod a+x /home/cloud-user/run-openshift-tests.sh'
+
+    - name: Copy openshift-tests binary to installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli scp '/tmp/kcli_${CLUSTER_NAME}_openshift-tests' \\
+            'cloud-user@${CLUSTER_NAME}-installer:/home/cloud-user/openshift-tests' \\
+          && kcli ssh 'cloud-user@${CLUSTER_NAME}-installer' 'chmod a+x /home/cloud-user/openshift-tests'
+
+    - name: Copy list of tests to installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli scp '/tmp/kcli_${CLUSTER_NAME}_${TEST_FILE}' \\
+            'cloud-user@${CLUSTER_NAME}-installer:/home/cloud-user/tests.txt' \\
+
+    - name: Fix permissions for kubectl on installer
+      ansible.builtin.shell:
+        cmd: kcli ssh 'root@${CLUSTER_NAME}-installer' 'chmod a+x /usr/bin/kubectl'
+
+    - name: Make kubeconfig available to non-root user on installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli ssh 'root@${CLUSTER_NAME}-installer' "sh -c '\\
+              cp /root/ocp/auth/kubeconfig /home/cloud-user/kubeconfig \\
+              && chown cloud-user /home/cloud-user/kubeconfig \\
+          '"
+
+    - name: Test kubeconfig for non-root user on installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli ssh 'cloud-user@${CLUSTER_NAME}-installer' "sh -c 'export KUBECONFIG=kubeconfig && oc get nodes'"
+
+    - name: Run script for openshift-tests binary on installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli ssh 'cloud-user@${CLUSTER_NAME}-installer' "/home/cloud-user/run-openshift-tests.sh"
+
+    - name: Compress results from openshift-tests binary on installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli ssh 'cloud-user@${CLUSTER_NAME}-installer' \\
+            "tar -czf openshift-tests.tar.gz e2e.log junit/ openshift-tests.log || { rm -f openshift-tests.tar.gz; exit 1; }"
+
+    - name: Copy results from installer to bastion host
+      ansible.builtin.shell:
+        cmd: |
+          kcli scp 'cloud-user@${CLUSTER_NAME}-installer:/home/cloud-user/openshift-tests.tar.gz' \\
+            '/tmp/kcli_${CLUSTER_NAME}_openshift-tests.tar.gz'
+
+    - name: Copy results from bastion host to artifacts directory
+      ansible.builtin.fetch:
+        src: '/tmp/kcli_${CLUSTER_NAME}_openshift-tests.tar.gz'
+        dest: '${ARTIFACT_DIR}/openshift-tests.tar.gz'
+
+    always:
```

Comment:
Do we need to this cleaning? Maybe worth to leave files for investigation or something, just a thought

Reply:
`openshift-tests` and `openshift-tests.tar.gz` are not particular lightweight, but if you prefer to keep it, i can comment it.